jvb/src/main/java/org/jitsi/videobridge/EndpointMessageTransport.java

```diff
@@ -157,6 +157,18 @@ public BridgeChannelMessage sourceVideoType(SourceVideoTypeMessage sourceVideoTy
         return null;
     }
 
+    @Override public BridgeChannelMessage receiverAudioSubscription(ReceiverAudioSubscriptionMessage receiverAudioSubscriptionMessage)
```

Comment:
Add a `@NotNull` annotation to the parameter to match the kotlin interface. Fix the line length (max 120 characters)